README.md

```diff
@@ -1,11 +1,8 @@
 This is a Chrome extension (written in C#) that shows a Xamarin.Forms page inside a popup. Xamarin.Forms and C# are compiled to WebAssembly via Uno.platform and Mono Wasm.
 
 ## How to build
-1. Open the App8.sln file with Visual Studio 2019 or VS for Mac 8.6.7 (or newer)
-1. Publish the .wasm project to a local folder
-1. Copy all files from browserExtensionAssets/ folder into it.
-1. Delete subfolders which start with underscore.
-1. Edit index.html to include `<link rel="stylesheet" type="text/css" href="./stylesheet.css" />`.
+1. Have .NET 3.1 SDK installed and accessible in your PATH
+1. Run `make.bat`
```

Comment:
as I mentioned before, don't forget to include a line that tells the user where is the generated folder and what's its name, given that this instructions now don't indicate how to specify it

Reply:
should be better now